docs/app/Examples/collections/Breadcrumb/Content/BreadcrumbIconDividerPropExample.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import { Breadcrumb } from 'stardust'
+
+export default class BreadcrumbIconDividerPropExample extends Component {
+
+  sections = [
+    { text: 'Home', link: true },
+    { text: 'Registration', link: true },
+    { text: 'Personal Information', active: true },
+  ];
+
+  render() {
+    return (
+      <Breadcrumb icon='right-angle' sections={this.sections} />
```

Comment:
Could we refactor the sections from the component instance to the global scope?  I think it helps illustrate that prop configured components are well suited to receive their values from outside the component:

```js
import React, { Component } from 'react'
import { Breadcrumb } from 'stardust'

const sections = [
  { text: 'Home', link: true },
  { text: 'Registration', link: true },
  { text: 'Personal Information', active: true },
]

export default class BreadcrumbDividerPropExample extends Component {
  render() {
    return (
      <Breadcrumb divider='/' sections={sections} />
    )
  }
}
```